ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -272,6 +274,12 @@ final class SandboxServer(
           )
       }
 
+      val seedService =
+        if (config.useSortableCid)
```

Comment:
Shouldn't it be the other way around? When `useSortableCid` is active, we create a seed service?

Reply:
You are right. Thanks.